### PR TITLE
More specification cleanup, Fixes #52, Fixes #53

### LINKED ISSUE #52: Complete section '2.b - Report by Exception' of the Sparkplug Spec
Additional work related to #32

### LINKED ISSUE #53: Complete section '2.d - Birth and Death Certificates' of the Sparkplug Spec
Additional work related to #32

### PR DESCRIPTION
**Pull request by wes-johnson**

### CODE REVIEW

**Review by nathandavenport — APPROVED** [review]

File: specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc [hunk 1]
```diff
@@ -12,20 +12,79 @@ _Sparkplug™ and the Sparkplug™ logo are trademarks of the Eclipse Foundation
...
+publish 'STATE' messages that represent its Birth and Death Certificates.*#
+
+[[conformance_sparkplug_secondary_host_application]]
+==== Sparkplug Secondary Host Application
+
+A Sparkplug Secondary Host Application it typically at a 'central location' and primarily receives
+data from multiple Sparkplug Edge Nodes. It also may send command messages to Sparkplug Edge Nodes
+to write to outputs at Sparkplug devices. A Secondary Host Application also sends rebirth requests
+to Edge Nodes when required.
+[tck-testable tck-id-conformance_primary_host]#[yellow-background]*Primary Host Application MUST
```
Comment (ibinshtok) [inline specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc:57]: Should it be: 'Secondary Host Application MUST NOT publish 'STATE' messages that represent its Birth and Death Certificates.'

File: specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc [hunk 2]
```diff
@@ -51,10 +51,10 @@ solutions.
 ==== Define the MQTT Payload
 
 Just as the MQTT Specification does not dictate any particular Topic Namespace, it also does it
-dictate any particular payload data encoding. The intent of the Sparkplug Specification is to strive
-to define payload encoding architectures that remain true to the original, lightweight, bandwidth
-efficient, low latency features of MQTT while adding modern encoding schemes targeting the
-SCADA/IIoT solution space.
+dictate any particular payload data encoding. The intent of the Sparkplug Specification is to define
```
Comment (ibinshtok) [inline specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc:54]: Should it be: 'Just as the MQTT Specification does not dictate any particular Topic Namespace, it also does not dictate any particular payload data encoding.' ?

File: specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc [hunk 3]
```diff
@@ -12,20 +12,79 @@ _Sparkplug™ and the Sparkplug™ logo are trademarks of the Eclipse Foundation
...
+A Sparkplug Edge Node is typically an 'Edge Gateway'. It sends and receives data to an MQTT Server
+using the spBv1.0/# namespace. Edge Nodes typically interact with physical devices to gather data
+and also write to device outputs.
 
 [[conformance_sparkplug_primary_host_application]]
 ==== Sparkplug Primary Host Application
- TODO: See Section 7 of http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
\ No newline at end of file
+
+A Sparkplug Primary Host Application it typically at a 'central location' and primarily receives
```
Comment (ckienle) [inline specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc:43]: Typo, change "it" to "is"

File: specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc [hunk 4]
```diff
@@ -12,20 +12,79 @@ _Sparkplug™ and the Sparkplug™ logo are trademarks of the Eclipse Foundation
...
+data from multiple Sparkplug Edge Nodes. It also may send command messages to Sparkplug Edge Nodes
+to write to outputs at Sparkplug devices. A Primary Host Application also sends rebirth requests to
+Edge Nodes when required.
+[tck-testable tck-id-conformance_primary_host]#[yellow-background]*Primary Host Application MUST
+publish 'STATE' messages that represent its Birth and Death Certificates.*#
+
+[[conformance_sparkplug_secondary_host_application]]
+==== Sparkplug Secondary Host Application
+
+A Sparkplug Secondary Host Application it typically at a 'central location' and primarily receives
```
Comment (ckienle) [inline specification/src/main/asciidoc/chapters/Sparkplug_10_Conformance.adoc:53]: Typo, change "it" to "is"

File: specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc [hunk 5]
```diff
@@ -51,10 +51,10 @@ solutions.
 ==== Define the MQTT Payload
 
 Just as the MQTT Specification does not dictate any particular Topic Namespace, it also does it
```
Comment (ckienle) [inline specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc:53]: change "it also does it" to "it also does not"

**Review by ibinshtok — APPROVED** [review]

**Pull request merged**